pyEpiabm/pyEpiabm/routine/file_population_config.py

```diff
@@ -180,7 +176,8 @@ def add_households(microcell: Microcell, household_number: int):
         person_index = 0
         for j in range(household_number):
             people_in_household = household_split[j]
-            new_household = Household(loc=microcell.location)
+            new_household = Household(microcell.cell, microcell,
+                                      loc=microcell.location)
```

Comment:
```suggestion
            new_household = Household(microcell,
                                      loc=microcell.location)
```

Could have this on one line or may not need to